src/test/java/org/fundacionjala/sfdc/stepdefinition/ContactSteps.java

```diff
@@ -0,0 +1,164 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+import org.fundacionjala.sfdc.pageobjects.Contacts.SFCDetailsPage;
+import org.fundacionjala.sfdc.pageobjects.Contacts.SFCMainPage;
+import org.fundacionjala.sfdc.pageobjects.Contacts.SFCNewModifyPage;
+import org.fundacionjala.sfdc.pageobjects.SalesForceContacts;
+import org.fundacionjala.sfdc.pageobjects.SalesForceMainTabClassic;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+import java.util.Properties;
+import java.util.StringJoiner;
+
+/**
+ * ContactSteps.java
+ * Class for steps of account.feature.
+ */
+public class ContactSteps {
+    private WebDriver webDriver;
+    private SalesForceContacts salesForceContacts;
+    private Properties propertiesManager;
+    private SFCDetailsPage contactsDetailPage;
+    private SFCMainPage contactsMainPage;
+    private SFCNewModifyPage contactsNewModifyPage;
+
+
+    /**
+     * Given step.
+     */
+    @Given("^Go to contacts section$")
+    public void goToContactsSection() {
+        System.out.println("This Step goes to contacts section");
+        SalesForceMainTabClassic salesForceMainTab;
+        salesForceMainTab = new SalesForceMainTabClassic("a.contactBlock", "li#AllTab_Tab");
+        salesForceMainTab.displayOptions();
+        salesForceMainTab.goToAccounts();
+
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I press new contacts button a new form is displayed$")
+    public void IPressNewContactsButtonANewFormIsDisplayed() {
+        System.out.println("This Step presses new account button");
+        contactsMainPage = new SFCMainPage();
+        contactsMainPage.clickNewButton();
+    }
+
+
+    /**
+     * And step.
+     */
+    @And("^I fill the required information and I press Save$")
+    public void IFillTheRequiredInformationAndIPressSave() {
+        System.out.println("This Step fills account name field");
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        contactsNewModifyPage = new SFCNewModifyPage();
+        contactsNewModifyPage.fillInfoNewContact(propertiesManager.getProperty("contactName"), propertiesManager.getProperty("contactLastName"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "contactLastName" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)